std/stdio.d

```diff
@@ -832,21 +832,19 @@ $(D rawRead) always reads in binary mode on Windows.
         return buffer;
     }
 
+    ///
     unittest
     {
-        static import std.file;
+        import std.file;
 
         auto deleteme = testFilename();
```

Comment:
@quickfur I renamed `deleteme` to `testFile` and also proposed an alternative PR (#4075) by putting a modified version into the docstring . Let me know your thoughts ;-)